buildman/manager/executor.py

```diff
@@ -390,17 +423,30 @@ def __init__(self, *args, **kwargs):
             "BUILDER_VM_CONTAINER_IMAGE", "quay.io/quay/quay-builder-qemu-coreos:stable"
         )
 
+    def validate(self, executor_config):
+        try:
+            create_job = self._build_request(executor_config, "POST", self._jobs_path(), json={})
+            if create_job.status_code != 400:
```

Comment:
This should check for `==` instead of `!=`. Also, still need that integer division to catch __all__ 4XX:
`if create_job.status_code // 100 == 4:`

Reply:
I think you might be misinterpreting the check here. The check is to see if we get a 400; if we do, then we know we have permission to start a Job, and it failed because we *purposefully* gave it an empty body. If we get a *non-400* (like a 401 or 403), we fail, because then we know we don't have permission.

Reply:
Ahh. Gotcha.